data/json/itemgroups/locations.json

```diff
@@ -83,6 +83,13 @@
       ["material_niter", 20],
       ["material_limestone", 5]
     ]
+  },{
+    "type":"item_group",
+    "id":"prison_textile",
+    "subtype": "collection",
+    "entries" : [
+      { "item": "blanket", "prob": 25 },
+      { "item": "jumpsuit", "prob": 35 }
```

Comment:
Should jumpsuit be replaced with striped_shirt and striped_pants? Prison wardens generally don't like inmates concealing stuff in ample pocket space.